app/src/views/Actors.vue

```diff
@@ -598,6 +628,36 @@ export default class ActorList extends mixins(DrawerMixin) {
       });
   }
 
+  async checkActorExist(name: string) {
+    const result = await ApolloClient.query({
+      query: gql`
+        query($query: ActorSearchQuery!, $seed: String) {
+          getActors(query: $query, seed: $seed) {
+            items {
+              ...ActorFragment
```

Comment:
Request only the name since the other properties won't be used

```suggestion
              name
```